README.md

```diff
@@ -59,6 +59,20 @@ urlpatterns = [
     
 ]
 ```
+* Configure cache in settings file
+```pythonstub
+CACHES = {
+    'default': {
+        'BACKEND': 'django.core.cache.backends.db.DatabaseCache',
+        'LOCATION': 'your_cache_table',
+    }
+}
+```
+
+* Creating the cache table
+```pythonstub
+python manage.py createcachetable
```

Comment:
what happens when we run this command multiple times?

Reply:
Asking because running this everytime before new deployment will be a pain

Reply:
No issues with this - if a table is already present then we'll get a message like 
`Cache table 'my_cache_table' already exists.`

We can just add `python manage.py createcachetable` in `run.sh` file

Reply:
cool